dkg/dkg.go

```diff
@@ -191,10 +201,17 @@ func Run(ctx context.Context, conf Config) (err error) {
 	// Write keystores, deposit data and cluster lock files after exchange of partial signatures in order
 	// to prevent partial data writes in case of peer connection lost
 
-	if err := writeKeystores(conf.DataDir, shares); err != nil {
-		return err
+	if conf.KeymanagerAddr != "" { // Save to keymanager
+		if err = writeKeysToKeymanager(ctx, conf.KeymanagerAddr, shares); err != nil {
+			return err
+		}
+		log.Debug(ctx, "Saved keyshares to keymanager")
```

Comment:
add URL ass param, suggest "import" as verb as that is what the keymannager API uses.

Reply:
<img width="1115" alt="Screenshot 2023-01-24 at 16 00 32" src="https://user-images.githubusercontent.com/43061995/214268938-5d5439d2-3b4b-46a5-95e5-286f21c35538.png">